src/pages/resultsView/mutation/ResultsViewMutationTable.tsx

```diff
@@ -105,5 +106,7 @@ export default class ResultsViewMutationTable extends MutationTable<IResultsView
         // customization for columns
         this._columns[MutationTableColumnType.EXON].render = 
             (d:Mutation[]) => (ExonColumnFormatter.renderFunction(d, this.props.genomeNexusCache, false));
+        this._columns[MutationTableColumnType.EXON].headerRender = 
+            () => <span style = {{display:'inline-block'}}>Exon <div>{this.props.totalExon} in total</div></span>;
```

Comment:
Is the `<div></div>` added to render the number in a new line?

Reply:
@onursumer yes, I change to <br/> instead